### PR TITLE
whatsnew - announcements

### PR DESCRIPTION
**Pull request by bjlittle**
### Description
This PR celebrates family... more over, welcoming and rejoicing new Iris core developers.

I think it's important to be very grateful for their time, efforts and contributions. So let's make sure to announce that to the community :grinning: 

---
[Consult Iris pull request check list]( https://scitools-iris.readthedocs.io/en/latest/developers_guide/pulls.html#the-iris-check-list)

### CODE REVIEW

File: docs/iris/src/whatsnew/latest.rst.template [hunk 1]
```diff
@@ -10,6 +10,12 @@ This document explains the changes made to Iris for this release
    :depth: 3
 
 
+Announcements
+=============
+
+* N/A
+
+
```
Comment (bjlittle) [inline docs/iris/src/whatsnew/latest.rst.template:18]: Added this new section to the `template`.

Ordering of sections is important, and IMHO `Announcements` should go first :wink:

File: docs/iris/src/whatsnew/latest.rst.template [hunk 2]
```diff
@@ -28,6 +34,12 @@ Incompatible Changes
 * N/A
 
 
+Deprecations
+============
+
+* N/A
+
+
```
Comment (bjlittle) [inline docs/iris/src/whatsnew/latest.rst.template:42]: Oopsie! This section was missing from the `template`.

**Review by trexfeathers — CHANGES_REQUESTED** [review]
The `Announcements` category also needs adding to the Contribution Categories in [whats_new_contributions.rst](https://github.com/SciTools/iris/blob/54cb982d0f7a734a8493756f6052992550009d3b/docs/iris/src/developers_guide/documenting/whats_new_contributions.rst)

**Review by trexfeathers — APPROVED** [review]

**Comment by trexfeathers** [comment]
Oh fudge I must have approved without merging. Sorry @bjlittle are you able to sort these conflicts fairly easily?

**Comment by bjlittle** [comment]
@trexfeathers Awesome sauce! :100:

**Pull request merged**